src/components/ProjectChatBox/Chat.tsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from 'react';
+
+import { Icons } from '../Icons';
+
+const Chat = () => {
+  const [message, setMessage] = useState<string>('');
```

Comment:
this state is not being used for anything right?

remove it until you add in real logic 🚮 
```suggestion
```